lib/tochtli/reply_queue.rb

```diff
@@ -56,7 +56,7 @@ def register_message_handler(message, handler=nil, timeout=nil, &block)
 
     def on_delivery(delivery_info, metadata, payload)
       class_name       = metadata.type.camelize.gsub(/[^a-zA-Z0-9\:]/, '_') # basic sanity
-      reply_class      = eval(class_name)
+      reply_class      = Object.const_get(class_name)
```

Comment:
@totrash: Maybe it's time to upgrade ? Ruby 1.9.3 was [released 4 years ago](https://www.ruby-lang.org/en/news/2011/10/31/ruby-1-9-3-p0-is-released/) and it's [not supported from February 2015](https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/).

How about this (ugly) solution ?
```ruby
  class_name.split('::').inject(Object) do |mod, class_name|
    mod.const_get(class_name)
  end
```
I think it's still better then `eval`.